packages/documentation-site/patternfly-docs/content/developer-resources/patternfly-accessibility.md

```diff
@@ -3,34 +3,35 @@ id: PatternFly's accessibility
 section: accessibility
 ---
 
-As a design system, PatternFly provides accessible building blocks to work with. We do our best to cover all areas that are within our control and take at least some of the accessibility work off your hands.
+# PatternFly components accessibility 
 
-With constant enhancements to PatternFly, we maintain this accessibility through a combination of automated and manual testing. We use [aXe: The Accessibility Engine](https://www.deque.com/axe/) on our build to ensure that all components pass this accessibility audit before they’re added to PatternFly. 
+We do our best to build accessibility into PatternFly's components to take some of the accessibility work off your hands.
 
-We regularly audit keyboard accessibility with both manual testing and integration tests. We’re also aiming to provide full support for Voice Over as our main screen reader, but we still test our components through NVDA and JAWS. As part of our manual audit, every component is run through Voice Over to make sure they’ll be as accessible as possible in your products.
+As PatternFly is updated and enhanced, we maintain  accessibility through a combination of automated and manual testing. We use [aXe: The Accessibility Engine](https://www.deque.com/axe/) to ensure that all components pass an accessibility audit before they’re added to PatternFly. 
 
-## What PatternFly aims to address
+We also regularly audit keyboard accessibility with both manual testing and integration tests. We aim to provide full support for Voice Over as our main screen reader, but we still test our components through NVDA and JAWS. As part of our manual audit, every component is run through Voice Over to make sure they’ll be as accessible as possible in your products.
 
-Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use PatternFly or contribute to PatternFly as a designer or developer, these are the items you can expect to be covered in PatternFly:
+## PatternFly accessibility standards
 
+Our goal is to meet [level AA in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa). If you use or contribute to PatternFly, these are the accessibility guidelines you can expect PatternFly to meet the following: 
```

Comment:
The second sentence sounds a bit off? wdyt of something like, "If you use or contribute to PatternFly, the following accessibility guidelines are some that you can expect PatternFly to meet:"